### PR TITLE
Describe how to add a new repo to our tenant

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

**Comment by jpopelka** [comment]
@FrNecas BTW the ticket for adding the specfile repo is [here](https://softwarefactory-project.io/r/c/config/+/23948)

File: README.md [hunk 1]
```diff
@@ -7,12 +7,34 @@
...
+a job which could be used by more repositories, you can add it to
+[jobs.yaml](https://github.com/packit-service/packit-service-zuul/blob/main/zuul.d/jobs.yaml).
+
+To enable Zuul in another repository, add it to
+[our tenant config](https://softwarefactory-project.io/cgit/config/tree/resources/tenant-packit-service.yaml)
+by following [The Config repository](https://softwarefactory-project.io/docs/user/config_repo.html)
+and [Contribute to a project](https://softwarefactory-project.io/docs/user/index.html).
+Specifically:
+
+- Add your ssh key to https://softwarefactory-project.io/r/settings/#SSHKeys if it's not already there.
```
Comment (FrNecas) [inline README.md:27]: Did you manage to actually make this work? Following the link after logging in through Github throws Forbidden.

Comment (jpopelka) [reply README.md:27]: hmm, I never got Forbidden in the webui, so you might be seeing a different issue after all

**Review by FrNecas — APPROVED** [review]
Thanks, it's clearer now.

File: README.md (hunk 1 above)
Comment (TristanCacqueray) [reply README.md:27]: It seems like there is an issue when creating new user in this gerrit. Perhaps adds the `softwarefactory-operations-team@redhat.com` point of contact for zuul and gerrit issues.

**Review by TristanCacqueray — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jpopelka) [reply README.md:26]: added, thanks Tristan

**Pull request merged**